src/utils/index.ts

```diff
@@ -11,4 +14,20 @@ const isPlatformAndroid = (): boolean => {
   return Platform.OS === "android"
 }
 
-export { DateTimeUtils, StorageUtils, isPlatformiOS, isPlatformAndroid }
+const displaySelfAssessment = (): boolean => {
+  return DISPLAY_SELF_ASSESSMENT === "true"
+}
+
+const displayNextSteps = (): boolean => {
+  // converting url string to boolean to make this more explicit and to please typescript
```

Comment:
What do you think about not having this comment?  It is extraneous as this is clear from the type signatures.